src/main/java/seedu/address/model/Model.java

```diff
@@ -45,4 +46,10 @@ void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
      */
     void updateFilteredPersonList(Predicate<ReadOnlyPerson> predicate);
 
+    /**
+     * Update a specified tag {@code oldTag} with a new tag {@code newTag}
```

Comment:
Coding standard violation:exclamation: The first statement of the header comment does not follow the prescribed style. See point 4, 3rd bullet point of the [[Java Coding Standard: Comments](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments)]